tests/fafnir_tests/mod.rs

```diff
@@ -0,0 +1,151 @@
+                    WHEN subclass IN ('post_box','post_office') THEN 'post'
+                    WHEN subclass IN ('cafe') THEN 'cafe'
+                    WHEN subclass IN ('school','kindergarten') THEN 'school'
+                    WHEN subclass IN ('alcohol','beverages','wine') THEN 'alcohol_shop'
+                    WHEN subclass IN ('bar','nightclub') THEN 'bar'
+                    WHEN subclass IN ('marina','dock') THEN 'harbor'
+                    WHEN subclass IN ('car','car_repair','taxi') THEN 'car'
+                    WHEN subclass IN ('hospital','nursing_home', 'clinic') THEN 'hospital'
+                    WHEN subclass IN ('grave_yard','cemetery') THEN 'cemetery'
+                    WHEN subclass IN ('attraction','viewpoint') THEN 'attraction'
+                    WHEN subclass IN ('biergarten','pub') THEN 'beer'
+                    WHEN subclass IN ('music','musical_instrument') THEN 'music'
+                    WHEN subclass IN ('american_football','stadium','soccer','pitch') THEN 'stadium'
+                    WHEN subclass IN ('art','artwork','gallery','arts_centre') THEN 'art_gallery'
+                    WHEN subclass IN ('bag','clothes') THEN 'clothing_store'
+                    WHEN subclass IN ('swimming_area','swimming') THEN 'swimming'
+                    WHEN subclass IN ('castle','ruins') THEN 'castle'
+                    ELSE subclass
+                END;
+            $$ LANGUAGE SQL IMMUTABLE;", &[]).unwrap();
+}
+
+pub fn main_test(es_wrapper: ElasticSearchWrapper, pg_wrapper: PostgresWrapper) {
+    init(&pg_wrapper);
+    let fafnir = concat!(env!("OUT_DIR"), "/../../../fafnir");
+    super::launch_and_assert(
+        fafnir,
+        vec![
+            "--dataset=test".into(),
+            format!("--connection-string={}", &es_wrapper.host()),
+            format!("--pg=postgres://test@{}/test", &pg_wrapper.host()),
+        ],
+        &es_wrapper,
+    );
+
+    // Test that the postgres wrapper contains 4 rows
+    let rows = &pg_wrapper.get_rows();
+    assert_eq!(rows.len(), 4);
+
+    // Test that the place "Ocean Studio" has been imported in the elastic wrapper
+    let pois: Vec<mimir::Place> = es_wrapper
+        .search_and_filter("Ocean Studio", |_| true)
+        .collect();
+    assert_eq!(&pois.len(), &1);
+
+    // Test that the place "Ocean Studio" is a POI
+    let ocean_place = &pois[0];
+    assert!(&ocean_place.is_poi());
+
+    // Test that the coord property of a POI has been well loaded
+    let ocean_poi = &ocean_place.poi().unwrap();
+    let coord_ocean_poi = &ocean_poi.coord;
+    assert_eq!(&coord_ocean_poi.lat(), &24.46275578041472);
+
+    // Test Label
+    let label_ocean_poi = &ocean_poi.label;
+    assert_eq!(label_ocean_poi, &"Ocean Studio");
+
+    // Test Properties
+    let properties_ocean_poi = &ocean_poi.properties;
+    let amenity_tag = properties_ocean_poi
+        .into_iter()
+        .filter(|p| p.key == "amenity")
```

Comment:
minor: I think `find` would be better there -no need for `collect` nor `pop`

Reply:
done